src/net/sourceforge/kolmafia/persistence/HolidayDatabase.java

```diff
@@ -294,31 +257,29 @@ public static final void setMoonPhases(final int ronaldPhase, final int grimaceP
 
       // Adjust the new year by the appropriate number of days.
 
-      HolidayDatabase.NEWYEAR += phaseError * MS_PER_DAY;
-      HolidayDatabase.BOUNDARY += phaseError * MS_PER_DAY;
-      HolidayDatabase.COLLISION += phaseError * MS_PER_DAY;
+      HolidayDatabase.NEWYEAR.plusDays(phaseError);
```

Comment:
Oh this is non mutating so I need to fix this tomorrow. Though I don't quite understand what's going on here anyway

Reply:
I do understand what's going on here now (there is a very helpful comment above! It seems that my changes make the first two points handled already. So now this code only exists for handling an incorrect system clock - with that in mind, am of a mind to remove this code. I'd be interested in hearing from other @kolmafia/staff, in particular @Veracity0

Reply:
I would be tempted to add a message to the console if the HolidayDatabase's calculated day number doesn't match with the daynumber reported by api.php and leave it at that. Or simply warn that you must have your system clock wrong if the moon phases are not what we expected!